chaincode/compute_plan_dag.go

```diff
@@ -67,7 +71,7 @@ func (dag *ComputeDAG) sort() error {
 		if ready {
 			final = append(final, current[i])
 			if _, ok := IDPresents[current[i].ID]; ok {
-				return fmt.Errorf("compute plan error, ID use twice: %s", current[i].ID)
+				return fmt.Errorf("Compute plan error: Duplicate training task ID: %s", current[i].ID)
```

Comment:
The error message should not start with a capital letter since it can be composed later on with another.

Reply:
I see @inalgnu already fixed it! Under what scenario would we compose this error with others?

Reply:
@AurelienGasser https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Thank you! 

Following that logic, what would you think of removing the "Compute plan error: " part of the error string?

Reply:
Talked with @thibaultrobert : let's leave it as it is now.